### PR TITLE
Add extension instructions in manifest

### PR DESCRIPTION
**Pull request by chuan9**
BUG=XWALK-6775

### CODE REVIEW

**Comment by chuan9** [comment]
@jondong @lisa0314 Please have a review, thanks

File: public/documentation/manifest/extensions_zh.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# 插件程序
+
+`xwalk_extensions`字段用于指定Crosswalk应用中的插件程序。如果您使用crosswalk-app-tools打包Crosswalk应用时，并且需要连同插件一块打包进去，那么需要在manifest文件中指定插件。
```
Comment (jondong) [inline public/documentation/manifest/extensions_zh.md]: 一块比较口语化，用一起是不是好一点

File: public/documentation/manifest/extensions_zh.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# 插件程序
```
Comment (jondong) [inline public/documentation/manifest/extensions_zh.md]: 这里我记得我们之前是统一用“扩展程序”这个名字的吧，因为插件听起来与Cordova plugin类似

**Comment by jondong** [comment]
@chuan9 some nits, thanks for the PR

**Comment by chuan9** [comment]
@jondong Done, thanks for your review

**Comment by jondong** [comment]
LGTM, thanks

**Pull request merged**